packages/core/chrome/core-chrome-browser-internal/src/ui/header/collapsible_nav.tsx

```diff
@@ -357,6 +356,14 @@ export function CollapsibleNav({
                 defaultMessage: 'Add integrations',
               })}
             </EuiButton>
+            <EuiScreenReaderLive focusRegionOnTextChange>
+              {i18n.translate('core.ui.primaryNav.screenReaderAnnouncement', {
+                defaultMessage: '{title}',
+                values: {
+                  title: appId,
+                },
+              })}
+            </EuiScreenReaderLive>
```

Comment:
This still can't live in the `EuiCollapsibleNav` because the nav is a flyout and flyouts are dynamically rendered. Live regions, especially ones announcing route changes, should be statically available on every page.

You'll need to move this region to the nav itself instead, i.e. https://github.com/elastic/kibana/blob/3c76b817ba749a32dc70d2522175b661263a4203/packages/core/chrome/core-chrome-browser-internal/src/ui/header/header.tsx#L170

Reply:
Ok, thank you! I think I'm following - is the announcement too short or is this a non-issue? 

https://user-images.githubusercontent.com/20343860/220386883-76197353-27fd-47b5-8677-5476715e4669.mp4